charts/fybrik/templates/fybrik-applications-rb.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.manager.enabled }}
+{{- if .Values.applicationNamespace}}
```

Comment:
I like this change and seems like it should work. Does require the application namespace to exist prior to installing Fybrik.  Bootstrap will do this when deploying using that tooling. If deploying manually should be documented.